Index: reghack-20130625/reghack.c
===================================================================
--- reghack-20130625.orig/reghack.c	2013-12-13 12:56:05.621871567 -0300
+++ reghack-20130625/reghack.c	2013-12-13 13:04:55.113885747 -0300
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
+#include <limits.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <string.h>
